utils/pubkeys.ts

```diff
@@ -0,0 +1,49 @@
+import { flow, pipe } from "fp-ts/lib/function";
+import * as TE from "fp-ts/TaskEither";
+import * as E from "fp-ts/Either";
+import { readableReportSimplified } from "@pagopa/ts-commons/lib/reporters";
+import * as RA from "fp-ts/ReadonlyArray";
+import { NewPubKeyPayload } from "../generated/definitions/internal/NewPubKeyPayload";
+import { JwkPubKeyHashAlgorithmEnum } from "../generated/definitions/internal/JwkPubKeyHashAlgorithm";
+import { calculateThumbprint } from "../utils/jose";
+import { AssertionRef } from "../generated/definitions/external/AssertionRef";
+
+export const MASTER_HASH_ALGO = JwkPubKeyHashAlgorithmEnum.sha512;
+
+/**
+ * Check if the input public key payload algo is the master one (current is sha512).
+ * If the check succeed, returns an array containing the input payload, othrwise returns an array containig both the input payload and the input payload with algo set to master.
+ *
+ * @param publicKeyPayload a public key payload containig an hash algorithm and a public key
+ * @returns
+ */
+export const pubKeyToAlgos = ({
+  algo,
+  pub_key
+}: NewPubKeyPayload): ReadonlyArray<NewPubKeyPayload> =>
```

Comment:
since this is a utility method we should avoid to accept an input that is valid only for reservation

Reply:
you are right! 
I defined a new type using ts-commons pub_key to use in utils in comit a62ef6cd7a2d362761fe830b4a5181624d51a2bc